testbenches/common/v/vanilla_core_stall.def

```diff
@@ -0,0 +1,27 @@
+// stalls that stall entire pipeline
+`VANILLA_CORE_STALL(stall_icache_store, stall_all & stall_icache_store, exe_pc)
+`VANILLA_CORE_STALL(stall_remote_ld_wb, stall_all & stall_remote_ld_wb, exe_pc)
+`VANILLA_CORE_STALL(stall_remote_flw_wb, stall_all & stall_remote_flw_wb, exe_pc)
```

Comment:
When there is a remote write back, the pc in exe might actually be invalid (exe has bubble), or some unrelated instruction that has nothing to do with the stall. The more accurate and relevant info would be the PC of the remote load, or the PC of the instruction that causing the contention in WB.

Reply:
k... so would i use the scoreboard tracker to track this? 
is this tracked correctly in the current tracer?

Reply:
so would this be the pc in the exe stage when a register is scored?

Reply:
I would benefit from more input on what pc should be shown here. It looks like I need to track pcs associated with idiv and imul instructions?

Reply:
k, i commented them out and left a TODO comment

Reply:
it's counted as an unknown stall at exe_pc

Reply:
OK. Works for me. 

Is there some message that says "unknown stalls were encountered, this may be because ..."?

Reply:
Not so easy to print it once. Not super hard to print it one per instance. 

I think it makes more sense to report the stall type with a not super useful pc than it does to report it as 'unknown.'
I'm happy to iterate on finding a better pc to report for wb stalls, but i think the merits of this PR easily meet the bar for merging. It's output is at least as correct as the current tracer's and it's 3x faster.

Finding a better pc for write back stalls seems to me like enough work to merit it's own PR.

Reply:
I guess my pc reporting comment was more directly in response to feedback from @tommydcjung. 

But following that feedback results in more of these 'unknown' stalls, so it's not unrelated to your suggestion of a print message. It sounds like you agree with me that reporting the stall type with the current exe_pc is more helpful than reporting unknown?

Reply:
Yes. I do want to know which PC caused it, but I think the data of where the stall happened is more useful. I don't know where the stall happened if I just know what PC caused it.

When I think of SGEMM, where these stalls happen, I want to answer the question: are my stalls happening at the tail of my FMA sequence? At the start? At the end? Those questions tell me something fundamental about read latencies in the network for a given access pattern and kernel. To me, that is actionable data.

Is the compiler spreading across ports correctly? What matters is that loads were coming back and conflicting with other writes to the register file. 

Using this information I could adjust the schedule and register allocation of my code. Potentially.

Reply:
We can add a caveat to the wb stalls that they are not necessarily caused by the PC that the stall is being attributed to. Instead, this PC is just is where it was observed.

Reply:
For posterity, in SGEMM at the moment, 10% of our stalls are these wb stalls. I would love more insight into why.

Reply:
I'm not saying we should report it as unknown. I think it's a good idea to report which remote load PC caused the stall, but  there is also a PC that is affected by it, which can be either exe_pc or exe_bubble_pc. I think remote load PC is more useful info for developers, but that could depend on situation.

Reply:
What I'm proposing is that we merge this PR reporting the stall types with exe_pc, and then I'll start a new branch to resolve this issue quickly (hopefully within a day or two). I know of a couple people that are waiting on this tool and I don't think this type of stall is a major concern for them.

Reply:
Let's address this in a different PR.